### PR TITLE
Next

### PR DESCRIPTION
**Pull request by adgad**
/cc @Financial-Times/next-myft @AlbertoElias @expe 

* Use SVGs instead of sprite (with PNG fallback)
* Tweaked padding/margins on icons for better touchability
* Added hover effect
* Mixin/class for vertical orientation of icons
* Added whatsapp share button

There will be new icons (circular) that will be uploaded to the ftsocial image set, that this (and indeed older versions of o-share) will use.

Note: I think this is a breaking change, mainly because of the extra spacing which could break older implementations

screen shot 2015-10-01 at 12 28 19
screen shot 2015-10-01 at 12 28 26

### CODE REVIEW

**Comment by AlbertoElias** [comment]
Maybe make the pr to the eslint branch to keep things clean?

File: demos/src/data.json [hunk 1]
```diff
@@ -11,6 +11,7 @@
     	"withGoogleplus": true,
     	"withReddit": true,
     	"withPinterest": true,
-    	"withLink": true
+    	"withLink": true,
+        "withWhatsapp": true
```
Comment (AlbertoElias) [inline demos/src/data.json]: Nicer to put it in the same order as the elements appear in on the spec markup (main.mustache)

**Comment by adgad** [comment]
Can do, or can merge the elint branch to master separately and then handle this PR?

**Comment by AlbertoElias** [comment]
just seems to make sense to make prs to the branch you branched of

*Alberto Elias*
Developer
FT Technology

+44 (0)74 75765609
Number One Southwark Bridge, London SE1 9HL


<https://www.facebook.com/financialtimes> <https://twitter.com/FT>
<http://www.linkedin.com/company/financial-times>
<https://plus.google.com/+FinancialTimes/posts>
<http://www.youtube.com/user/FinancialTimesVideos>


On 1 October 2015 at 13:04, Arjun Gadhia <notifications@github.com> wrote:

> Can do, or can merge the elint branch to master separately and then handle
> this PR?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Financial-Times/o-share/pull/48#issuecomment-144707339>
> .
>

-- 

------------------------------
This email was sent by a company owned by Pearson plc, registered office at 
80 Strand, London WC2R 0RL.  Registered in England and Wales with company 
number 53723.

**Comment by adgad** [comment]
Okay, will do that

**Pull request closed without merging**